doc/ChangeLog

```diff
@@ -1,4 +1,267 @@
+
+
+Contributors
+------------
+
+
+Significant changes to scientifically-supported configurations since ctsm5.2.0
+------------------------------------------------------------------------------
+
+Notes of particular relevance for users for ctsm5.2.0 to ctsm5.3.0
+------------------------------------------------------------------
+
+Caveats for users (e.g., need to interpolate initial conditions):
+
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+
+Changes made to namelist defaults (e.g., changed parameter values):
+
+Changes to the datasets (e.g., parameter, surface or initial files):
+
+Changes to documentation:
+
+Things being deprecated (which will be removed):
+
+Notes of particular relevance for developers for ctsm5.2.0 to ctsm5.3.0:
+------------------------------------------------------------------------
+
+Caveats for developers (e.g., code that is duplicated that requires double maintenance):
+
+Changes to tests or testing:
+
+
+Testing summary: release testing (regular ctsm_sci fates mosart rtm mksurfdata_esmf python)
+----------------
+ [PASS means all tests PASS; OK means tests PASS other than expected fails.]
+
+  build-namelist tests (if CLMBuildNamelist.pm has changed):
+
+    derecho -
+
+  python testing (if python code has changed; see instructions in python/README.md; document testing done):
+
+    derecho -
+
+  regular tests (aux_clm: https://github.com/ESCOMP/CTSM/wiki/System-Testing-Guide#pre-merge-system-testing):
+
+    derecho -----
+    izumi -------
+
+  fates tests: (give name of baseline if different from CTSM tagname, normally fates baselines are fates-<FATES TAG>-<CTSM TAG>)
+    derecho -----
+    izumi -------
+
+  any other testing (give details below):
+
+  mksurfdata_esmf
+     derecho ---- run "make all" to create all datasets (completes in XXX hours)
+     (crop-global-SSP2-4.5-ne30 was longest at 6 hr, most completed in XXX hours)
+
+  ctsm_sci
+     derecho ----
+
+  mosart
+     derecho ----
+     izumi ------
+
+  rtm
+     derecho ----
+
+
+Answer changes ctsm5.2.0 to ctsm5.3.0
+-------------------------------------
+
+Changes answers relative to baseline: Yes! New surface datasets so answers change for ALL simulations!
+
+  Summarize any changes to answers, i.e.,
+    - what code configurations: ALL
+    - what platforms/compilers: ALL
+    - nature of change (roundoff; larger than roundoff/same climate; new climate):
+      new climate
+
+   If this tag changes climate describe the run(s) done to evaluate the new
+   climate (put details of the simulations in the experiment database)
+       - casename:
+
+   URL for LMWG diagnostics output used to validate new climate:
+
+
+Other details
+-------------
+
+List any git submodules updated (cime, rtm, mosart, cism, fates, etc.):
+
+===============================================================
+===============================================================
+Tag name: ctsm5.2.029
+Originator(s): multiple
```

Comment:
we should include the list from the #2500 PR.

Reply:
@lawrencepj1 @ekluzek @wwieder @adrifoster @samsrabin @billsacks @lifang0209 @linniahawkins @olyson